core/domain/classifier_services.py

```diff
@@ -305,9 +313,9 @@ def _update_classifier_training_job(classifier_training_job_model, status):
     classifier_training_job_model.put()
 
 
-def save_classifier_training_job(algorithm_id, exp_id, exp_version,
-                                 state_name, status, training_data,
-                                 job_id="None"):
+def save_classifier_training_job(algorithm_id, exp_id,
+                                 exp_version, state_name, training_data,
+                                 status='NEW', job_id='None'):
```

Comment:
Do not use default values for status and job_id. Specify them explicitly.

Also, have NEW defined somewhere as a named constant (probably CLASSIFIER_JOB_STATUS_NEW and similarly for the others) and refer to that instead.